torch/csrc/jit/backends/backend_detail.cpp

```diff
@@ -161,8 +189,14 @@ Module codegen_backend_module(
             def __setstate__(self, state):
                 self.__method_compile_spec = state[0]
                 self.__processed_module = state[1]
-                self.__create_backend()
-                self.__handles = self.__backend.compile(self.__processed_module, self.__method_compile_spec)
+                # state[2] indicates wherethercreate_backend
```

Comment:
Rephrase the comments? The two lines on state[2]

Reply:
Yeah I suggested deleting the first line (seems unfinished)